python/turbodbc_test/test_has_arrow_support.py

```diff
@@ -0,0 +1,26 @@
+from mock import patch
+
+from turbodbc.cursor import _has_arrow_support
+
+import six
+import pytest
+
+# Skip all parquet tests if we can't import pyarrow.parquet
+pa = pytest.importorskip('pyarrow')
```

Comment:
Yes, with this statement the tests are skipped if `pyarrow` is not installed, this is needed e.g. for the unit tests on windows.